_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe } from 'node:test';
+import { test, type Page } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist } from './helpers/helpers';
+
+describe('Popular Repositories', () => {
+  test('Clean - Delete any current repos that exist', async ({ page }) => {
+    await deleteAllRepos(page);
+  });
+
+  test('Navigate to Popular Repos tab, assert the Add button is displayed', async ({ page }) => {
```

Comment:
Similarly here. It is rather simple test - does it need to be separate?

Reply:
No, just learning

Reply:
This may be a good spot to try the "test.step" method that @dominikvagner  introduced.

Reply:
Ok, now using `test.step`

Reply:
I had to revert that use of `test.step`, test looses session and then cannot find repos to remove.

Reply:
`test.step()` should not impact the session, playwright uses one for the whole test. 
it's use is for adding a step to the test report and for us "documenting" what the test does in more detail, so it most likely wasn't caused by the step func 💭😅